### DISCUSSION TITLE
Raw images transverse view of Plus volume reconstruction

### DISCUSSION THREAD

**Discussion by Karl-Philippe** [PlusToolkit/PlusLib#931] [Q&A] [answered]
Hello,

I work with tracked ultrasound and use Plus to acquire and reconstruct volumetric ultrasound using the VolumeReconstruction tool included in Plus. I was able to recontruct a volume without errors (note, there are some space between the ultrasound slices) but to segment the volume, I need to see the raw images "in plane" (as with the cross-sectional view of the cross-sectional driver) but the cross-sectional driver does not provide a variable transformation to see all the images in plane. 

Is there any way to see (scroll through) the raw images in a volume reconstruction?

Thank you,

**Comment by ungi** [comment, marked as answer]
Data visualization is more convenient to do in 3D Slicer, not PLUS. Recorded tracked ultrasound sequences can be imported in Slicer, the same volume reconstruction algorithm that PLUS uses is also available in 3D Slicer (SlicerIGSIO extension). And the SlicerIGT extension contains modules that can visualize tracked ultrasound (2D) overlaid with the reconstructed volume (3D). All of these features are shown e.g. in this demo: https://youtu.be/WyscpAee3vw

**Reply by Karl-Philippe** [reply]
Perfect, this is what I was looking for! Thanks @ungi!

**Reply by Karl-Philippe** [reply]
Hi @ungi,

I followed the demo, but this is still not exaclty what I was looking for, I am able to visualise the tracked ultrasound overlaid with the reconstructed ultrasound but I cannot segment the reconstructed volume from the tracked ultrasound (2D). I am looking for to visualise the volume in the orrientation of the transducer.
Is this possible?
The closer I got to solve exaclty this problem is by, after reconstruction, using transverse mode the volume reslice driver in the orientation of the live replay of the tracked ultrasound (and by ajusting the right ImageToVolume transform),
Thank you,
![image](https://user-images.githubusercontent.com/47655918/162216469-c6bb9ba7-b86a-477a-9c15-5133bc9f586e.png)

**Reply by ungi** [reply]
Did you try to select the original ultrasound image as the background for any of the 2D views? In your screenshot, the selector for the red slice is dropped down. It says "Live_reconstruction" now. Can you choose the original ultrasound there? By the way, if you drop down the double arrow on the left, that section will show more options. E.g. you can show the original and reconstruction images overlaid. Although if you do that, it looks best if you give them different colors in the Volumes module.
